docs/docusaurus/docs/guides/expectations/profilers_data_assistants_lp.md

```diff
@@ -1,18 +1,17 @@
 ---
-sidebar_label: 'Profilers and Data Assistants'
-title: 'Profilers and Data Assistants'
-id: profilers_data_assistants_lp
-description: Use the Onboarding Data Assistant and Profiler to Profile your data and create Expectation Suites.
+sidebar_label: 'Data Assistants'
+title: 'Data Assistants'
+id: data_assistants_lp
```

Comment:
```suggestion
id: data_assistants_lp
```
NOTE: When you make changes like this, build tests will fail because you didn't rename the referenced file to match. I've implemented the necessary changes to resolve any build test failures.